src/Symfony/Component/Security/Acl/Dbal/Schema.php

```diff
@@ -61,94 +77,11 @@ public function addToSchema(BaseSchema $schema)
     }
 
     /**
-     * Adds the class table to the schema.
-     */
-    protected function addClassTable()
-    {
-        $table = $this->createTable($this->options['class_table_name']);
-        $table->addColumn('id', 'integer', array('unsigned' => true, 'autoincrement' => 'auto'));
-        $table->addColumn('class_type', 'string', array('length' => 200));
-        $table->setPrimaryKey(array('id'));
-        $table->addUniqueIndex(array('class_type'));
-    }
-
-    /**
-     * Adds the entry table to the schema.
-     */
-    protected function addEntryTable()
```

Comment:
@stof I like this code because I think that is good and cohesive. I should reduce the quality to the 2.3 and create a new PR for future versions (3.0 maybe) ?